biz.aQute.bndlib/src/aQute/bnd/build/Workspace.java

```diff
@@ -816,6 +816,13 @@ public RepositoryPlugin getRepository(String repo) {
 
 	public void close() {
 		cache.remove(getPropertiesFile().getParentFile().getParentFile());
+
+		try {
+			super.close();
+		}
+		catch (IOException e) {
+			/* Swallow exception for BC */
```

Comment:
BC = Backwards Compatibility

It means that `Workspace.close()` did not throw an exception before but Processor.close() is declared to throw an `IOException`. I'm open to suggestions on how to make this better.